apps/dashboard/src/components/SandboxDetailsSheet.tsx

```diff
@@ -62,6 +63,16 @@ const SandboxDetailsSheet: React.FC<SandboxDetailsSheetProps> = ({
     return getRelativeTimeString(sandbox.updatedAt)
   }
 
+  const copyToClipboard = async (text: string) => {
```

Comment:
Can we extract to somewhere more common?

We also have a copy button for api key dialogs